src/v2i-hub/package_plugins.sh

```diff
@@ -23,6 +23,8 @@ PLUGINS=(
 
 # package up all plugins
 for PLUGIN in "${PLUGINS[@]}"; do
+    # remove the link if it exists already
+    rm -f "$PLUGIN"/bin
```

Comment:
This line can be avoided by adding `--force` to `ln` command on the next line

Reply:
Changed to use --force on ln